tgui/packages/tgui/interfaces/XenomorphExtractor.jsx

```diff
@@ -0,0 +1,140 @@
+import { useBackend, useLocalState } from '../backend';
+import { Section, Button, Stack, NoticeBox, LabeledList, Flex, Box, Dropdown, Divider } from '../components';
+import { Window } from '../layouts';
+
+export const XenomorphExtractor = (_props, context) => {
+  const { act, data } = useBackend(context);
```

Comment:
```suggestion
export const XenomorphExtractor = () => {
  const { act, data } = useBackend();
```
Context is depreciated and not used in react > 16. Also no reason to use props if you aren't going to be using props.

Reply:
Im back!!!!